test/Microsoft.AspNet.Mvc.FunctionalTests/Compiler/Resources/MvcTagHelpersWebSite.MvcTagHelper_Home.CreateWarehouse.html

```diff
@@ -6,7 +6,7 @@
 <form action="/MvcTagHelper_Home/CreateWarehouse" method="post">        <div>
             <label class="warehouse" for="City">City</label>
             <input size="50" type="text" data-val="true" data-val-minlength="The field City must be a string or array type with a minimum length of &#39;2&#39;." data-val-minlength-min="2" id="City" name="City" value="" />
-            <span class="field-validation-valid" data-valmsg-for="City" data-valmsg-replace="true"></span>
+            <span class="field-validation-valid" data-valmsg-for="City" data-valmsg-replace="true" />
```

Comment:
we already have <span></span> case in many places, I just picked this test to simulate an invalid but possible case.